flexmeasures/cli/utils.py

```diff
@@ -59,3 +63,44 @@ def get_command(self, ctx, cmd_name):
             click.echo(click.style(self.deprecation_message, fg="red"), err=True)
             ctx.invoked_default = self.default_cmd_name
         return super().get_command(ctx, cmd_name)
+
+
+def get_timerange_from_flags(
+    start: Optional[datetime], end: Optional[datetime], timezone, **kwargs
+) -> Tuple[Optional[datetime], Optional[datetime]]:
+
+    flags = ["last_hour", "last_day", "last_week", "last_month", "last_year"]
+
+    # if any of the flag in `flags` is passed in kwargs
+    if len([kwarg for kwarg in kwargs if kwarg in flags]) > 0:
+        end = datetime.now(tz=timezone).replace(microsecond=0, second=0, minute=0)
```

Comment:
This variable should be called `current_hour`, so that it's easier to see how you are using it.

Reply:
Yes, that's clearer.